api/types/types.go

```diff
@@ -142,25 +142,25 @@ func (n *NodeRule) Validate() error {
 
 // PacketInjection packet injector API parameters
 type PacketInjection struct {
-	BasicResource
-	Src              string
-	Dst              string
-	SrcIP            string
-	DstIP            string
-	SrcMAC           string
-	DstMAC           string
-	SrcPort          int64
-	DstPort          int64
-	Type             string
-	Payload          string
+	BasicResource    `yaml:",inline"`
+	Src              string `yaml:"Src"`
+	Dst              string `yaml:"Dst"`
+	SrcIP            string `yaml:"SrcIP"`
+	DstIP            string `yaml:"DstIP"`
+	SrcMAC           string `yaml:"SrcMAC"`
+	DstMAC           string `yaml:"DstMAC"`
+	SrcPort          int64  `yaml:"SrcPort"`
+	DstPort          int64  `yaml:"DstPort"`
+	Type             string `yaml:"Type"`
+	Payload          string `yaml:"Payload"`
 	TrackingID       string
```

Comment:
Any reason to not put a tag for TrackingID ?

Reply:
It is generated from server side, so i skipped a tag